src/Realm.C

```diff
@@ -2520,8 +2520,11 @@ Realm::compute_vrtm(const std::string& velName)
   auto vrtm = fieldMgr.get_field<double>(
     get_field_ordinal(*metaData_, velName + "_rtm"));
 
+  auto* vrtm_field = metaData_->get_field<VectorFieldType>(
+    stk::topology::NODE_RANK, velName + "_rtm");
```

Comment:
Does this potentially need a 'sync_to_host' or anything like that?

Reply:
@alanw0 I don't think so. `vrtm_field` is only used for `stk::mesh::selectField` here and not really modified.